### PR TITLE
Remove view docs dropdown

### PR DESCRIPTION
**Pull request by stsewd**
I'll continue @ericholscher's work here

It looks like this

screenshot_2019-03-05 anonymoususer s profile read the docs
screenshot_2019-03-05 home read the docs

### CODE REVIEW

File: readthedocs/templates/core/project_list_detailed.html [hunk 1]
```diff
@@ -29,24 +29,15 @@
...
-    </span>
-    {% else %}
     <ul class="module-item-menu">
-      <li><a href="{{ project.get_builds_url }}">{% trans "No Docs" %}</a></li>
+      <li>
+        {% if project.has_good_build %}
+          <a href="{{ project.get_docs_url }}">{% trans "View Docs" %}
+          {% else %}
+            <a href="{{ project.get_builds_url }}">{% trans "No Docs" %}</a>
+          {% endif %}
```
Comment (agjohnson) [inline readthedocs/templates/core/project_list_detailed.html]: nit: `else` and `endif` indentation is wrong.

Comment (stsewd) [reply readthedocs/templates/core/project_list_detailed.html]: Actually it was missing a `</a>` :/

**Comment by humitos** [comment]
> but we should return to this and potentially remove the link to the build list page when a project doesn't have a build

What would be the solution here? Removing the "No docs" button completely and only show "View docs" when there is docs to show?

**Comment by stsewd** [comment]
> but we should return to this and potentially remove the link to the build list page when a project doesn't have a build

I guess we can address that in the site redesign

**Pull request merged**